src/tests/commonTests.ts

```diff
@@ -910,13 +910,20 @@ export async function floatingUIOwner(
  * Helper to test t9n component setup
  *
  * @param {TagOrHTML|TagAndPage} componentSetup - A component tag, html, or an object with e2e page and tag for setting up a test
+ * @param intlProps
+ * @param nonIntlProps
  */
-export async function t9n(componentSetup: TagOrHTML | TagAndPage): Promise<void> {
+export async function t9n(
+  componentSetup: TagOrHTML | TagAndPage,
+  intlProps = true,
```

Comment:
This signature might cause issues for components that have both intl props in addition to extra ones. I suggest dropping `intlProps` (always testing for those) and changing `nonIntlProps` to `extraMessageProps` and testing for those if a value was indeed passed along. We can revisit this if we come up with a consistent way to group the non-intl ones (e.g., a11y labels).

Reply:
make sense. proposing `additionalMessageProps` inplace of `extraMessageProps`.  WDYT ?